sentry-android-core/src/test/java/io/sentry/android/core/ContextUtilsUnitTests.kt

```diff
@@ -161,4 +166,28 @@ class ContextUtilsUnitTests {
         val memInfo = ContextUtils.getMemInfo(mock(), logger)
         assertNull(memInfo)
     }
+
+    @Test
+    fun `registerReceiver calls context_registerReceiver without exported flag on API 32-`() {
+        val buildInfo = mock<BuildInfoProvider>()
+        val receiver = mock<BroadcastReceiver>()
+        val filter = mock<IntentFilter>()
+        val context = mock<Context>()
+        whenever(buildInfo.sdkInfoVersion).thenReturn(Build.VERSION_CODES.S)
+        ContextUtils.registerReceiver(context, buildInfo, receiver, filter)
+        verify(context).registerReceiver(eq(receiver), eq(filter))
+    }
+/*
+    context.registerReceiver(BroadcastReceiver, IntentFilter, int) throws with NoSuchMethod exception in tests
+    Re-enable this test when possible
```

Comment:
this is probably because Robolectric is running it with API 32 or lower by default? You can probably try to set the SDK version for this specific test:
```
@Config(sdk=[33])
@Test
...
```

Reply:
Was looking for this, thanks!